wiki/Storyboards/en.md

```diff
@@ -10,3 +10,17 @@ A **storyboard** (SB) is a custom-made animated background that accompanies a [b
 Storyboards are stored in beatmap folders as either standalone [`.osb` files](/wiki/osu!_File_Formats/Osb_(file_format)) or extensions to the `[Events]` section of a [`.osu` file](/wiki/osu!_File_Formats/Osu_(file_format)). Because of this, it's possible to create different storyboards for different maps within a [beatmapset](/wiki/Glossary#beatmapset).
 
 Storyboards are often very difficult to make and require considerable time and expertise in standard animation and in the creation of graphics. osu! offers a [design editor](/wiki/Beatmap_Editor/Design) inside of the [beatmap editor](/wiki/Beatmap_Editor) to aid the creation of storyboards, but most avid storyboard producers opt to instead program via [storyboard scripting](/wiki/Storyboard_Scripting) directly. Many creators choose to write programs in full-featured programming languages to generate storyboards scripts, because complex visual effects can require a great amount of storyboard code to produce.
+
+## Storyboarding
+
+*See also: [Storyboarding](/wiki/Storyboarding).*
+
+## Beatmaps with nice storyboard
```

Comment:
- "nice" is subjective and not formal
- listing a small amount of random maps (not to mention the list is pretty dated, and more modern examples should be on it) doesn't help the article in general (nor this specific section). I kind of get the idea, but it is outside of this article's scope, please undo the change